canonical-web-and-design/charmhub.io
# Make charmhub specific config, make app.py more generic

## Done

- Added `charmhub.py` config
- Updated details endpoint (now takes any string that matches the regex and a `type` param)
- Rename `detail_type` to `entity_type` to make it more general
- Rename `base_index.html` to `base_layout.html`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure homepage works as before
- Visit http://0.0.0.0:8045/anystring see the charm details page
- Visit http://0.0.0.0:8045/anystring?type=bundle see the bundle details page
